examples/complex/i18n.js

```diff
@@ -13,6 +13,6 @@ module.exports = {
     prefix: '${',
     suffix: '}',
   },
-  loadLocaleFrom: (locale, namespace) =>
-    import(`./src/translations/${namespace}_${locale}`).then((m) => m.default),
+  loadLocaleFrom: async (locale, namespace) =>
+    require(`./src/translations/${namespace}_${locale}`),
```

Comment:
For this we need to find a better way before 2.0 to fix this issue: https://github.com/aralroca/next-translate/issues/851